mods/farming/api.lua

```diff
@@ -140,6 +140,7 @@ farming.place_seed = function(itemstack, placer, pointed_thing, plantname)
 
 	local under = minetest.get_node(pt.under)
 	local above = minetest.get_node(pt.above)
+	local place_to = {x = pt.above.x, y = pt.above.y, z = pt.above.z}
```

Comment:
Surely this is equivalent to:
`local place_to = pt.above`
?

Reply:
Not my code xD
Done!